### ISSUE TITLE
Using "url" as the name for an "other text input field" breaks things!

### ISSUE THREAD

**Issue by adrianbj** [justb3a/processwire-imageextra#31]
It stalls uploads from finalizing, and if you reload the page, the thumbnail will be blank. Removing it from the list gets things working again. I am assuming that other PW image field properties like:
filename, width, height, etc will also be problematic. Perhaps you can note that these aren't allowed and prevent them from being created if they are accidentally entered?

**Comment by hubertgrzesiek** [comment]
I have to agree with @adrianbj. Just spent last 30 minutes trying to figure out what happened. Thankfully I found this comment :)
